Core/Object Arts/Dolphin/MVP/Presenters/Tree/UI.TreePresenter.cls

```diff
@@ -9,7 +9,7 @@ UI.SelectableItemsPresenter subclass: #'UI.TreePresenter'
 UI.TreePresenter guid: (Core.GUID fromString: '{87b4c688-026e-11d3-9fd7-00a0cc3e4a32}')!
 UI.TreePresenter comment: 'TreePresenter implements a <treePresenter> component that can be used to display the contents of a <treeModel> within a <treeView>. This is similar to the way a tree view is used to display a file system''s contents in the Window''s explorer. For presentation of a tree as a hierarchy of nested "folders" see the <FolderPresenter> class.
 
-TreePresenter conforms to <selectableItems> to provide methods for accessing the current selection. In addition, the <treePresenter> protocol allows the tree to be navigating using the #expand: , #collapse: and #up methods.
+TreePresenter conforms to <selectableItems> to provide methods for accessing the current selection. In addition, the <treePresenter> protocol allows the tree presentation to be modified using the #expand: , #collapse: and #up methods.
```

Comment:
I think the original text is more correct if "navigating" is replaced with "navigated"